test/integration.spec.js

```diff
@@ -68,3 +70,20 @@ test('integration : babel syntax error', t => {
     ''
   ]);
 });
+
+test('integration : webpack multi compiler', t => {
+
+  const syncWebpackWithPlugin = deasync(function(config, fn) {
+    const compiler = webpack(config);
+    // Plugin must be applied to `MultiCompiler` rather than via webpack.config.
+    compiler.apply(new FriendlyErrorsWebpackPlugin());
+    compiler.run(fn);
+    return compiler;
+  });
+
+  var logs = output.captureLogs(() => {
+    syncWebpackWithPlugin(require('./fixtures/multi-compiler/webpack.config'));
+  });
+
+  assert(/DONE  Compiled successfully in /.test(logs));
```

Comment:
Thank you for adding this test!
Having not worked with multi-compiler, it is not clear to me how would this fail if we hadn't included your modification.
Would it display the message twice?